bigtable-hbase-parent/bigtable-hbase-1.x/src/main/java/com/google/cloud/bigtable/hbase1_x/BigtableAdmin.java

```diff
@@ -91,7 +91,13 @@ public void enableTableAsync(TableName tableName) throws IOException {
     throw new UnsupportedOperationException("getCompactionStateForRegion");
   }
 
-  /** {@inheritDoc} */
+  /**
+   * {@inheritDoc}
+   *
+   * The snapshot will be created with the ttl configured by
+   * {@link com.google.cloud.bigtable.hbase.BigtableOptionsFactory#BIGTABLE_SNAPSHOT_DEFAULT_TTL_SECS_KEY}
+   * key in the configuration. Will default to the serverside default.
```

Comment:
Not sure if we will default ttl to the serverside default under what condition. Perhaps "If not configured, the ttl will be set to serverside default"?